site/sidebar.js

```diff
@@ -230,6 +230,17 @@ module.exports = {
                 "mfa/about",
                 "mfa/errors",
                 "mfa/rateLimits",
+                {
+                    type: "category",
+                    label: "Callbacks",
```

Comment:
```suggestion
                    label: "Wehbooks",
```
We call this section webhooks in the rest of the documentation - id like to remain consistent if possible

Reply:
pushed up a fix, let me know if it looks good